appveyor.yml

```diff
@@ -12,7 +12,7 @@ build: off
 
 environment:
   global:
-    MAVEN_OPTS: "-Xmx1536M"
+    MAVEN_OPTS: "-Xmx2048M"
```

Comment:
It's probably best that we don't do this in multiple PRs.  It's an experiment where I'm doing it, and it hasn't finished running yet.